ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -54,6 +58,7 @@ export class Pay extends IronfishCommand {
     let fee = flags.fee as unknown as number
     let to = flags.to
     let from = flags.account
+    const message = flags.memo as string
```

Comment:
this already comes in as undefined | string so you don't need to typecast it

Reply:
fixed.